tests/virt_autotest/vgpu.pm

```diff
@@ -0,0 +1,368 @@
+# SUSE's openQA tests
+# Copyright 2020 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: A test to pass an NVIDIA vGPU to guest via libvirt. Only KVM hosts are supported.
+# Test environment: one AMPERE GPU card, such A10, A30 etc., with SR-IOV capability and/or MIG features in host machine;
+#                   A UEFI guest vm is defined in the host, and they are ssh accessible from host.
+# Test flow:
+#    - install the NVIDIA vGPU manager on host and reboot
+#    - create a GPU Instance and a Compute Instance if MIG feature is available
+#    - create a vGPU device in host
+#    - assign the vGPU device to guest vm
+#    - install the vGPU driver in guest vm
+#    - detach vGPU from guest and remove the vGPU
+# Maintainer: qe-virt@suse.de, Julie CAO <JCao@suse.com>
+
+use base "virt_feature_test_base";
+use strict;
+use warnings;
+use utils;
+use testapi;
+use virt_autotest::common;
+use version_utils qw(is_sle);
+use virt_autotest::utils;
+use Utils::Backends qw(use_ssh_serial_console is_remote_backend);
+use ipmi_backend_utils qw(reconnect_when_ssh_console_broken);
+
+our $log_dir = "/tmp/vgpu";
+our $vm_xml_save_dir = "/tmp/download_vm_xml";
```

Comment:
Will above two var be used across test files? If not, maybe `local` is better to constrain scope.

Reply:
I can't quite distinguish the symbols of these variables, I just referenced them from other test pm files. These variables should be limited in within this file only. so let me change it to `local`.

Reply:
when I changed `our` to `local`:
```
 28 local $log_dir = "/tmp/vgpu";
 29 local $vm_xml_save_dir = "/tmp/download_vm_xml";
```

Test failed :face_exhaling: 
```
[2024-08-02T03:43:49.305805Z] [warn] [pid:12107] !!! autotest::loadtest: error on tests/virt_autotest/vgpu.pm: Global symbol "$log_dir" requires explicit package name (did you forget to declare "my $log_dir"?) at /var/lib/openqa/pool/8/os-autoinst-distri-opensuse/tests/virt_autotest/vgpu.pm line 28.
  Global symbol "$vm_xml_save_dir" requires explicit package name (did you forget to declare "my $vm_xml_save_dir"?) at /var/lib/openqa/pool/8/os-autoinst-distri-opensuse/tests/virt_autotest/vgpu.pm line 29.
```

Reply:
I had a try, and yes it will give the warning when "use strict;use warnings", but fine without using those.

Besides, I had a check again -- local is more suitable to override the upper layer variable from this point down to the deepest call. So not very suitable for the case. Please ignore my comment. Sorry :(. Using global will be good enough.